enclave/cert.c

```diff
@@ -235,6 +235,9 @@ static mbedtls_x509_crt* _find_root_cert(mbedtls_x509_crt* chain)
     if (!_x509_buf_equal(&p->subject_raw, &p->issuer_raw))
         return NULL;
 
+    if (p == NULL)
+        OE_TRACE_ERROR("root cert was not found\n");
+
```

Comment:
Test for p being NULL can be moved before line 235 as the call to x509_buf_equal will fault if p is NULL.